test-status/api.js

```diff
@@ -114,9 +115,8 @@ function createReportedCheckParams(
           'resulted in test breakage or fix the broken tests.\n\n' +
           'If you believe that this pull request was not the cause of this ' +
           'test breakage (i.e., this is a flaky test) please contact the ' +
```

Comment:
I think we should change the messaging here so that skipping a test is the last thing to recommend, after
1. restarting the Travis job or build
2. rebasing
3. if all else fails then contact build cop to skip test or advise

Reply:
Done. wdyt?

Reply:
Looks awesome